packages/docusaurus/src/client/baseUrlIssueBanner/BaseUrlIssueBanner.tsx

```diff
@@ -80,7 +80,8 @@ function BaseUrlIssueBannerEnabled() {
   // useLayoutEffect fires before DOMContentLoaded.
   // It gives the opportunity to avoid inserting the banner in the first place
   useLayoutEffect(() => {
-    window[InsertBannerWindowAttribute] = false;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    (window as any)[InsertBannerWindowAttribute] = false;
```

Comment:
I find it a bit clearer / readable with something like `// @ts-expect-error: not worth typing this`

Reply:
Funny, when I remove the eslint-disable, typescript is not complaining anymore